clicmd

```diff
@@ -1,12 +1,68 @@
 #!/bin/bash -eu
 
+#
+# YADRO OpenBMC Command Line Interface
+# Copyright (C) 2020-2021 YADRO
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+
 # Base CLI command handler.
 # This script is used for setting up the environment, elevating privileges
 # and handling hierarchical commands.
 
 # prevent modification of executable path
 PATH="/bin:/sbin:/usr/bin:/usr/sbin"
 
+# Role related definitions
+ROLE_ADMIN="admin"
+ROLE_OPERATOR="operator"
+ROLE_USER="user"
+
+GROUP_ADMIN="%GROUP_ADMIN%"
+GROUP_OPERATOR="%GROUP_OPERATOR%"
+GROUP_USER="%GROUP_USER%"
+
+# If we are running escalated, we have GROUP already set by the caller
+if [[ "${USER}" != "root" || ! -v GROUP ]]; then
+  if [[ "${USER}" = "root" ]]; then
+    echo "Can't run CLI commands as root directly, use an admin account" >> &2
```

Comment:
It leads to an error:
```
root@vegman-sx20-QEMU-BMC:~# bmc help
/usr/bin/bmc: line 29: syntax error near unexpected token `&'
```

Reply:
Typo. Will fix. It must be `>` as well, not `>>`.